Alloy/commands/purgetss/index.js

```diff
@@ -0,0 +1,50 @@
+const { exec } = require('child_process');
+const commandExistsSync = require('command-exists').sync;
+
+module.exports = async function(args, program) {
+	// console.log('module:', program.module);
+	// console.log('vendor:', program.vendor);
+	let options = '';
+	try {
+		if (args.length === 0) {
+			if (program.all) options += ' --all';
+			if (program.debug) options += ' --debug';
+			execCommand(`purgetss ${options}`);
+		} else {
+			args.forEach(command => {
+				switch (command) {
+					case 'init':
+						execCommand('purgetss init');
+						break;
+					case 'build':
+						execCommand('purgetss build');
+						break;
+					case 'watch':
+						execCommand('purgetss watch');
+						break;
+					case 'fonts':
+						if (program.vendor) options += ` -v=${program.vendor}`;
+						if (program.modules) options += ' --modules';
+						execCommand(`purgetss fonts${options}`);
+						break;
+					case 'build-fonts':
+						if (program.modules) options += ' --modules';
+						execCommand(`purgetss build-fonts${options}`);
+						break;
+					case 'module':
+						execCommand('purgetss module');
+						break;
+				}
+			});
+		}
+	} catch (error) {
+		//
```

Comment:
You can use `die()` from utils here:
https://github.com/tidev/alloy/blob/5e9e25e58ce0fac3e8c12e7583297a5f64d27101/Alloy/utils.js#L521